package_up

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+zip_filename=instacart-pp-$(whoami).zip
```

Comment:
👍 on the `whoami` .. 
I have been adding bonus points for candidates adding their names to the archive

I have just tested this command with my last candidate, it worked like a charm !